konfera/models/order.py

```diff
@@ -24,7 +24,7 @@
 
 
 class Order(KonferaModel):
-    price = models.DecimalField(decimal_places=2, max_digits=12, validators=[MinValueValidator(0)])
+    price = models.DecimalField(decimal_places=2, max_digits=12, validators=[MinValueValidator(0)], default=0)
```

Comment:
I don't like this default value. *Explicit is better than implicit* especially when we are talking about money. :-)